spec/spec_helper.rb

```diff
@@ -1,7 +1,11 @@
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
+require 'combustion'
 require 'stitch_fix/log_weasel'
+
+Combustion.initialize! :active_support
```

Comment:
From the Combustion README:
> Combustion is a library to help you test your Rails Engines in a simple and effective manner, instead of creating a full Rails application in your spec or test folder.

> It allows you to write your specs within the context of your engine, using only the parts of a Rails app you need.

More on the gem [here](https://github.com/pat/combustion). 

It actually worked quite well, and after adding it I immediately reproduced the error Dean ran into while upgrading `stitchfix-api_client` in Kingmob.